noir-projects/noir-contracts/contracts/test/test_contract/Nargo.toml

```diff
@@ -7,5 +7,5 @@ type = "contract"
 [dependencies]
 aztec = { path = "../../../../aztec-nr/aztec" }
 address_note = { path = "../../../../aztec-nr/address-note" }
-value_note = { path = "../../../../aztec-nr/value-note" }
+uint_note = { path = "../../../../aztec-nr/uint-note" }
```

Comment:
I changed `TestContract` to use `UintNote` instead of `ValueNote` because only `UintNote` supports partial notes.